site/source/partials/header.hbs

```diff
@@ -27,14 +27,41 @@
   <!-- stylesheet -->
   <link rel="stylesheet" href="{{assets}}css/style.css">
 
-  <!-- Cedar -->
-
-  <!-- esri Cedar -->
-  <script type="text/javascript" src="http://square.github.io/crossfilter/d3.v3.min.js"></script>
-  <script type="text/javascript" src="{{assets}}js/vega.js"></script>
```

Comment:
I think we can get rid of the locally hosted copy of vega.js now. LMK if you want me to do that and update the grunt build, etc.